eslint.config.js

```diff
@@ -251,6 +251,7 @@ module.exports = [
     },
     files: ['public/app/features/alerting/**/*.{ts,tsx,js,jsx}'],
     rules: {
+      'sort-imports': ['error', { ignoreDeclarationSort: true }],
```

Comment:
We could probably use this rule everywhere, but I didn't want to go crazy and open up everyone to more likely conflicts :D